test/failure_test.js

```diff
@@ -37,31 +37,4 @@ suite('Known failures', function() {
     assert.equal(subject.getParameter("cn"), "Z\\;");
     assert.equal(subject.getFirstValue(), "mailto:z@example.org");
   });
-
-  // Quoted multi-value parameters leak into the value
-  // Please see https://github.com/kewisch/ical.js/issues/634
-  testKnownFailure('with quoted multi-value parameter', function() {
```

Comment:
Shouldn't this test move to the normal testsuite with the bug been fixed?

Reply:
Agree on this, let's move it to `parse_test.js`